app/utils/HeadersHelper.scala

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import controllers.documentation.Router
+import controllers.documentation.ReverseRouter
+import play.api.mvc.RequestHeader
+
+/**
+ * Helpers for returning additional links.
+ */
+object HeadersHelper {
+
+  /**
+   * Returns the canonical URL that shows the latest version of documentation of a page.
+   */
+  def canonical(page: String)(implicit rh: RequestHeader): String = {
+    // https://github.com/playframework/playframework.com/issues/43
+    val (lang, _) = Router.findLanguageAndVersionPath(rh.path)
+    val relativePath = ReverseRouter.latest(lang, page)
+    // May be relative per https://tools.ietf.org/html/rfc6596#section-3
+    relativePath
+  }
```

Comment:
The problem with this is that not every page exists in the latest version of the documentation, which means the link could 404.  Giving the search engines a 404 for the canonical version could well be worse than giving them no canonical link.

The documentation controller can and should work out a canonical link, if one exists, based on the data.  This logic will probably go best in the documentation actor.  There already exists logic for working out if a page exists for a particular version.